app/src/test/java/ch/epfl/favo/user/UserUnitTests.java

```diff
@@ -102,17 +103,19 @@ public void userCanAcceptOrRequestAtMostOnce() {
   }
 
   @Test
-  public void userNameIsValid() {
-    String username = TestConstants.USERNAME;
-    String pw = TestConstants.PASSWORD;
-    assertThrows(
-        NotImplementedException.class,
-        new ThrowingRunnable() {
-          @Override
-          public void run() throws Throwable {
-            UserUtil.getSingleInstance().createAccount(username, pw);
-          }
-        });
+  public void userSuccessfullyPostsToDB() {
+    CollectionWrapper mock = Mockito.mock(CollectionWrapper.class);
+    Mockito.doNothing().when(mock).addDocument(any(User.class));
+
+    String name = TestConstants.NAME;

```

Comment:
We can extract these into the FakeItemFactory. (Next sprint)

Reply:
sounds good